mlrun/api/schemas/__init__.py

```diff
@@ -99,7 +99,6 @@
     GrafanaStringColumn,
     GrafanaTable,
     GrafanaTimeSeriesTarget,
-    Metric,
```

Comment:
why?

Reply:
Because the `Metric` object has been removed and replaced by the new metrics dictionary (`ModelEndpointStatus.metrics`).